docs/UserGuide.md

```diff
@@ -149,8 +151,8 @@ Finds students whose names contain any of the given keywords.
 
 Format: `find NAME [MORE_NAMES]`
 
-* The search is case-insensitive.
-* Only the name is searched.
+* The search is case-insensitive (i.e. John, JOHN, john are all treated the same way).
+* You can only search for a student by their name.
```

Comment:
Good that this is pointed out because "find" command's parameters weren't clear initially